docs/resources/Emoji.md

```diff
@@ -11,7 +11,7 @@
 | --------------- | ---------------------------------------------------------------- | ------------------------------------------------------------------------- |
 | id              | ?snowflake                                                       | [emoji id](#DOCS_REFERENCE/image-formatting)                              |
 | name            | ?string (can be null only in reaction emoji objects)             | emoji name                                                                |
-| roles?          | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids | roles this emoji is whitelisted to                                        |
+| roles?          | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids | roles this emoji is delimited to                                        |
```

Comment:
i would say "limited to" or "locked to" for these

Reply:
Changing wording like this is more than simple `s/word1/word2/g` job, as I'm not sure this particular word can even be used like this (though I am not a native speaker so do not quote me on that). The statement would, in my opinion, sound better if phrased as "roles allowed to use this emoji" or something to that effect in this particular instance.

Reply:
Maybe "array of IDs corresponding to roles allowed to use this emoji. If not provided, then all roles can use this.", just to be clearer on the behaviour, perhaps?